tests/visco_plastic_phases_viscosity_jump/solution/solution-00000.0000.vtu

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" ?> 
```

Comment:
Please remove this file from the PR. The output should only be the gnuplot output.